api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -114,4 +115,24 @@ public void sendMessage_shouldSendMessage() throws Exception {
 		}
 	}
 	
+	@Test
+	public void sendMessage_shouldSaveTheSentMessage() throws MessageException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/openmrs/openmrs-core/file/2318749994/issues/source?bid=3204099&fileBranchId=3204129#l119)